tests/test_action_checks.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Initialize error counter
+errors=0
+
+# Function to simulate the "Check if review is requested" step
+check_review_requested() {
+  PR_TITLE="$1"
+  
+  if [[ "$PR_TITLE" =~ ^(no|skip)(-|\s)?review|skip(-|\s)?code(-|\s)?review|otter(ai)?(-|\s)?skip|otter(-|\s)?restricted ]]; then
+    echo "🦦 No review requested, skipping code review"
+    return 0
+  else
+    echo "🔍 Code review requested"
+    return 0
+  fi
+}
+
+# Function to simulate the "Check if PR is merged" step
+check_pr_merged() {
+  PR_STATE="$1"
+  
+  if [[ "$PR_STATE" == "merged" ]]; then
+    echo "🦦 PR is merged, skipping code review"
```

Comment:
🦦 Ensure that the messages are consistent in tone and style across the script for better readability.